src/widgets.ts

```diff
@@ -83,6 +83,22 @@ export abstract class Widget<
     return dispatchWidgetPostMessageEvent
   }
 
+  /**
+   * Public method to communicate with iframe widget about navigation events
+   * Can be called when the host has a 'back' event happen.
+   * This will send a post message event to the iframe widget, which if it's
+   * listening for the 'mx/navigation' event, it can make its changes and then
+   * send its own post message event back to the SDK for the 'onNavigation' callback
+   */
+  navigateBack() {
+    const iframeElement = this.iframe.contentWindow
+    const data = { mx: true, type: "mx/navigation", payload: { action: "back" } }
+
+    if (iframeElement) {
+      iframeElement.postMessage(data, "*")
```

Comment:
Since we are abstracting the postmessage details from the host here, without a new widget option to provide us the post message `targetOrigin`, I'm putting a * in. I know this isn't good practice, so if you have any suggestions I'd love to hear it.

Reply:
I don't think I understand the first part of your message. Why do we have to do this?

Reply:
In this flow, we are having the SDK send Connect the post messages, so the host doesn't have to think or know about it. But as a result, the host cannot control what the post message `targetOrigin` is, unless we expose another widget option to pass into this. So I'm just using the * right now. Sorry, did that help clear it up any?

Reply:
The old widget loader exposed a target origin for post messaging the config to connect.

```javascript
    var baseUrlPattern = /^https?:\/\/[^\/]+/i;
    var widgetBaseUrl = options.url && options.url.match(baseUrlPattern) ? options.url.match(baseUrlPattern)[0] : null;

    this.options.postMessageOrigin = options.postMessageOrigin || widgetBaseUrl || 'https://widgets.moneydesktop.com';
```

I'd prefer that we don't expose a config to the clients, since they shouldn't have to care, so maybe we do:

```javascript
    var baseUrlPattern = /^https?:\/\/[^\/]+/i;
    var widgetBaseUrl = options.url && options.url.match(baseUrlPattern) ? options.url.match(baseUrlPattern)[0] : null;

    this.options.postMessageOrigin = widgetBaseUrl || 'https://widgets.moneydesktop.com';
```

Reply:
That's not a bad idea, I'll try that out. Thanks!